### PR TITLE
Initial version of netflix dataset inclusion

### PR DESCRIPTION
**Pull request by jeffbanks**
Includes updates of Netflix experience for the NodeJS quick start.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,109 +1,178 @@
-# DataStax Node.js Driver for Apache Cassandra Quickstart
+# DataStax NodeJS Quickstart using Netflix
```
Comment (csplinter) [inline README.md]: Should we use the same title format as the Java sample?

"DataStax Desktop - Node.js Netflix example"

File: app.js [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+  const selectAll = 'SELECT * FROM ' + tableName + ' ALLOW FILTERING;'
+  console.log('Selecting all from Table: %s', tableName)
+  return client.execute(selectAll)
+
+}
+
+async function readAllInPrimaryByTitle(titleName) {
+
+  const selectAll = 'SELECT * FROM ' + TABLE_NETFLIX_PRIMARY + ' WHERE title = ? ALLOW FILTERING;'
```
Comment (csplinter) [inline app.js]: shouldn't need ALLOW FILTERING here and we definitely don't want to showcase that to the user

**Review by csplinter — APPROVED** [review]
Once we update the title to match that in the [Java example](https://github.com/DataStax-Examples/quickstart-java/tree/quickstart-java-netflix#datastax-desktop---java-netflix-example), +1 here

File: README.md [hunk 3]
```diff
@@ -1,109 +1,178 @@
...
+* Use the Netflix show dataset as example information across three differently constructed tables.
+* Observe how the partition key along with clustering keys produce an optimized experience.
+* Have fun!
 
 ## Project Layout
 
-* [quickstart.js](src/quickstart.js) - main application file with space to fill in CRUD operation code
-* [users.cql](users.cql) - Use this file to create the schema 
+* [app.js](app.js) - main application file
+* [netflix-shows](netflix-shows.cql) - foundation CQL for reference or use
```
Comment (jgillenwater) [inline README.md:19]:
```suggestion
* [netflix-shows](netflix-shows.cql) - file to create the schema
```

Changed for consistency across examples.

File: README.md [hunk 4]
```diff
@@ -1,109 +1,178 @@
...
 
-## Objectives
+## How this works
+To get started, read through the comments in the `app.js` to familiarize yourself with the steps for interacting with your own Cassandra database. The functions invoked by the `app.js` are created to provide more flexibility for modifications as you learn.
```
Comment (jgillenwater) [inline README.md:22]:
```suggestion
To get started, read the `app.js` comments to learn the steps for interacting with your Cassandra database. The functions invoked by the `app.js` are created to provide more flexibility for modifications as you learn.
```

File: README.md [hunk 5]
```diff
@@ -1,109 +1,178 @@
...
 
-* To demonstrate how to perform basic CRUD operations with the DataStax Node.js Driver.
-* The intent is to help users get up and running quickly with the driver. 
+## Setup and running
 
-## How this Sample Works
-This project walks through basic CRUD operations using Cassandra. The demo application will first insert a row of user data, select that same row back out, update the row and finally delete the user. The README includes the code snippets to be filled in to the main application code to complete the functionality.
+### Prerequisites
+If using [DataStax Desktop](https://www.datastax.com/blog/2020/05/learn-cassandra-datastax-desktop), no prerequisites will be required. The Cassandra instance is provided with the DataStax Desktop stack as part of container provisioning.
```
Comment (jgillenwater) [inline README.md:27]:
```suggestion
If using [DataStax Desktop](https://www.datastax.com/blog/2020/05/learn-cassandra-datastax-desktop), no prerequisites are required. The Cassandra instance is provided with the DataStax Desktop stack as part of container provisioning.
```

File: README.md [hunk 6]
```diff
@@ -1,109 +1,178 @@
...
 
-## Prerequisites
-  * A running instance of [Apache Cassandra®](http://cassandra.apache.org/download/) 1.2+
+If NOT using DataStax Desktop, spin up your own local instance of Cassandra exposing its address and port to align with the settings in the **app.js** file.  You will need to install and perform the following steps:
```
Comment (jgillenwater) [inline README.md:29]:
```suggestion
If NOT using DataStax Desktop, spin up your own local instance of Cassandra exposing its address and port to align with the settings in the `app.js` file.  You will need to install and perform the following steps:
```

File: README.md [hunk 7]
```diff
@@ -1,109 +1,178 @@
...
 
-### INSERT a user
-```javascript
-function insertUser(lastname, age, city, email, firstname) {
-  // TO DO: execute a prepared statement that inserts one user into the table
-  const insert = 'INSERT INTO users (lastname, age, city, email, firstname) VALUES (?,?,?,?,?)';
-  const params = [ lastname, age, city, email, firstname ];
-  return client.execute(insert, params, { prepare : true });
-}
+Simply kickoff the app from the command line.
```
Comment (jgillenwater) [inline README.md:49]:
```suggestion
Start the app from the command line.
```

**Review by jgillenwater — COMMENTED** [review]
Minor changes.

**Pull request merged**